java/code/src/com/redhat/rhn/domain/contentmgmt/SoftwareProjectSource.java

```diff
@@ -72,6 +72,24 @@ public void setChannel(Channel channelIn) {
         channel = channelIn;
     }
 
+    /**
+     * Gets the source Label.
+     *
+     * @return source label
+     */
+    public String sourceLabel() {
```

Comment:
Same as before, according to default naming, this should be `getSourceLabel`, no?